复线命令:
```json
/home/tianmai/workspace/tmp/test1 out/test3_gdb13/crashes/crashes/crash_000019

```
分析结果:
```json

Description: 不可信指令执行
Hash: 93d48e3d6d7a661889e7ed5f3d6eb8d2.93d48e3d6d7a661889e7ed5f3d6eb8d2
Exploitability Classification: 可利用的

```
栈回溯:
```json

#0  0x000055555555b000 in ?? ()
#1  0x0000555555555332 in testBadInstruction () at test.c:18 objfile: /home/tianmai/workspace/tmp/test1 offset: 0x1332
#2  0x0000555555555924 in main (argc=2, argv=0x7fffffffdd38) at test.c:299 objfile: /home/tianmai/workspace/tmp/test1 offset: 0x1924

```
崩溃点汇编:
```json

   0x55555555531f <testbadinstruction+65>:	mov    esi,0xff
   0x555555555324 <testbadinstruction+70>:	mov    rdi,rax
   0x555555555327 <testbadinstruction+73>:	call   0x555555555170 <memset@plt>
   0x55555555532c <testbadinstruction+78>:	mov    rax,qword ptr [rbp-0x8]
=> 0x555555555330 <testbadinstruction+82>:	call   rax
   0x555555555332 <testbadinstruction+84>:	mov    eax,0x0
   0x555555555337 <testbadinstruction+89>:	leave
   0x555555555338 <testbadinstruction+90>:	ret
   0x555555555339 <testabortsignal>:	endbr64

```
崩溃点源代码:
```json
test.c:18

13	int testBadInstruction() {
14	    char *a = valloc(64);
15	    mprotect(a, 64, PROT_READ | PROT_WRITE | PROT_EXEC);
16	    voidfn bad_function = (voidfn)a;
17	    memset(a, 0xff, 64);
18	    bad_function();   <-<
19	    return 0;
20	}
21	
22	void testAbortSignal() {

```
